pom.xml

```diff
@@ -2680,6 +2676,26 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>1.4.1</version>
+                <executions>
+                    <execution>
+                        <id>enforce-maven</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireMavenVersion>
+                                    <version>3.5.0</version>
```

Comment:
I'm using 3.3.9. Do you really need to enforce 3.5.0?

Reply:
Of course not, What version would be ok for the team ?

Reply:
3.3.9 is the last of the 3.3.x branche, and almost 2 years old, so maybe we can define it as a minimum.